ci/check.sh

```diff
@@ -44,10 +44,9 @@ if [[ "$1" == "--check" ]]; then
 fi
 
 banner "Executing in conda environment ${CONDA_DEFAULT_ENV} in directory ${root}"
+run "Style Checking" "ruff format fgpyo"
+run "Linting"        "ruff check --fix fgpyo"
 run "Unit Tests"     "python -m pytest -vv -r sx fgpyo"
-run "Import Sorting" "isort --force-single-line-imports --profile black fgpyo"
-run "Style Checking" "black --line-length 99 $black_extra_args fgpyo"
```

Comment:
It looks like `black_extra_args` is either now superfluous or should be repurposed